cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/eventData/startedCheckout.js

```diff
@@ -12,6 +13,7 @@ function getData(currentBasket) {
 
     var data = {};
     var basketItems = currentBasket.getProductLineItems().toArray();
+    var reconstructCartItems = [];
     // Create some top-level event data
```

Comment:
Are these comments intended to guide the integrator for customizations? If not, can they be removed?

Reply:
was just left in during earlier development on this project. will remove.